src/redux/categories/categories.js

```diff
@@ -0,0 +1,21 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] I kindly suggest that you follow the [ducks](https://github.com/erikras/ducks-modular-redux) pattern 

Wrong :x: 
`const CHECK_STATUS = 'CHECK_STATUS';`

Right :heavy_check_mark: 
`const CHECK_STATUS = bookstore/categories/CHECK_STATUS`